composer.json

```diff
@@ -56,7 +58,8 @@
         ],
         "phpunit": "phpunit",
         "ci-phpunit": "phpunit --coverage-clover=coverage.clover",
-        "cs-check": "php-cs-fixer fix --dry-run --diff",
+        "ci-phpstan": "phpstan analyse --no-progress -l7 src tests",
```

Comment:
It's better to use current directory (`.`). Filters should be declared in phpstan configuration